src/components/header/TopNavBar.jsx

```diff
@@ -38,37 +34,58 @@ class TopNavBar extends Component {
       >
         {/*Nike logo*/}
         <Pane className="logo">
-          <a href="/"><img src={nike} alt=""/></a>
+          <a href="/">
+            <img src={nike} alt="" />
+          </a>
         </Pane>
 
         {/*Category & Subcategory*/}
         <Pane className="commonCategory">
           <TabNavigation
             style={{
               textAlign: "center",
-              paddingLeft: "30px"
+              paddingLeft: "30px",
             }}
           >
             {this.state.categories.map((category) => (
               <Pane className="subnav">
                 <Tab className="top-navbar" key={category.id}>
-                  <NavLink exact to = {{pathname: category.url}} 
-                  style={{color: "#000", textDecoration: 'none'}}>{category.title}</NavLink>
+                  <NavLink
+                    exact
+                    to={{ pathname: category.url, headdata: category.title }}
+                    style={{ color: "#000", textDecoration: "none" }}
```

Comment:
why is styling necessary here?

Reply:
Nav link is a different component inside the tab component that we used before which doesn't necessarily inherit the styling from its previous components that's why upon experimenting used inline styling to make the component look good in our webpage. In my coding, you can see sometimes see that I have used some inline styling when the styling is limited to 2/3 elements rather than calling a class name here.

Reply:
as our guru @Richie017 said styling should always be in different file and to avoid inline styling even if it's only one element